### DISCUSSION TITLE
How to specify the default drivers directory

### DISCUSSION THREAD

**Discussion by Skypekey** [dbeaver/dbeaver#12499] [Q&A] [answered]
I want to specify the default driver directory in the portable version, but i don't know how can I configure it. I need some suggestions to solve this problem
The version is dbeaver-ce-21.0.0-win32.win32.x86_64

**Comment by uslss** [comment]
You can learn more about it [here](https://github.com/dbeaver/dbeaver/wiki/Admin-Manage-Drivers)

**Comment by Skypekey** [comment]
I read this document and i know how to configure this, but when i configured dbeaver,ini to specify data directory,  it is not  working properly. My configuration is
```
- data
datadir
```

**Comment by uslss** [comment, marked as answer]
@Skypekey default drivers location could be set in Preferences
![image](https://user-images.githubusercontent.com/31996417/116536388-8363bf80-a8ed-11eb-899c-91acf4406d1f.png)
